libkbfs/block_retrieval_worker_test.go

```diff
@@ -89,7 +89,12 @@ func (bg *fakeBlockGetter) getBlock(ctx context.Context, kmd KeyMetadata, blockP
 func (bg *fakeBlockGetter) assembleBlock(ctx context.Context,
 	kmd KeyMetadata, ptr BlockPointer, block Block, buf []byte,
 	serverHalf kbfscrypto.BlockCryptKeyServerHalf) error {
-	return errors.New("Not implemented")
+	source, ok := bg.blockMap[ptr]
```

Comment:
Somewhat cheating to avoid having to pass test crypto stuff to the fake block getter.